Tests/DependencyInjection/DoctrineExtensionTest.php

```diff
@@ -975,8 +976,9 @@ public static function cacheConfigurationProvider(): array
 
     public function testShardManager(): void
     {
-        $container = $this->getContainer();
-        $extension = new DoctrineExtension();
+        $container    = $this->getContainer();
+        $extension    = new DoctrineExtension();
+        $managerClass = 'Fake\\ShardManager';
```

Comment:
Sorry, one more. I missed this in my last review

Reply:
Not sure what's here, did you want me to use a real class and reference with `::class` ?

Reply:
yes